examples/metrics-server-operator/api/v1alpha1/groupversion_info.go

```diff
@@ -0,0 +1,36 @@
+/*
+Copyright 2020 TODO(akihito): assign copyright.
```

Comment:
I think The Kubernetes Authors is standard?

Reply:
Thank you for comment. I've fixed these.